src/ConfigDefinition.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ProcessorIconv;
+
+use Keboola\Component\Config\BaseConfigDefinition;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+class ConfigDefinition extends BaseConfigDefinition
+{
+    protected function getParametersDefinition(): ArrayNodeDefinition
+    {
+        $parametersNode = parent::getParametersDefinition();
+        // @formatter:off
+        /** @noinspection NullPointerExceptionInspection */
+        $parametersNode
+            ->children()
+                ->scalarNode('source_encoding')->isRequired()->end()
+                ->scalarNode('target_encoding')->defaultValue('UTF-8')->end()
+                ->scalarNode('modifier')
+                    ->defaultValue("")
+                    ->validate()
+                        ->ifNotInArray(['//TRANSLIT', '//IGNORE', ''])
```

Comment:
pokud je na modifier vlastní konfigurační property, dal bych jim nějaký lidský názvy (`ignore`, `transliterate`) a konvertoval to v procesoru